### ISSUE TITLE
No module named models.IALUT.IALUT_transform.IALUT._ext

### ISSUE THREAD

**Issue by TwilightArchon** [Wenhao-Li-777/FastLLVE#4]
![image](https://github.com/Wenhao-Li-777/FastLLVE/assets/134197144/ad7605bb-6d71-4b0d-8b62-5786d6cc5f86)
I have tried many methods to build the setup.py, but it never worked. Can you please provide with the missing _ext file? Thank you!

**Comment by Wenhao-Li-777** [comment]
The _ext file is built during the installation of the extension. So you may need to solve the problem of not being able to install the extension rather than find the generated _ext file. Can you describe what issues you encountered while installing the extension?

**Comment by TwilightArchon** [comment]
Yes, i tried to run the model, but it says there is no module named xxxx.IALUT._ext. I checked the folder, and indeed, the ext file was not there. I searched up how to solve this issue, and the answer is to  build setup.py. And the issue I'm encountering is there seems to be incompatible versions of cuda and something which is very difficult to resolve. So I'm wondering can you post the .ext file somewhere to directly download? Thank you!
